### PR TITLE
Tech lead responsibilities doc

### PR DESCRIPTION
**Pull request by sharonwarner**
### Description
This PR adds a document detailing tech leads' responsibilities.

### CODE REVIEW

File: docs/tech-lead-description.md [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+* Balances the team's work on new product requirements with technical debt/refactoring work to ensure engineering quality
+* Works with product and design to create tickets for the engineering team
+* Monitors risks to the Caseflow team's mission and handles or escalates to the appropriate place where necessary, including:
+  * Risks that team projects will not be able to meet their requirements or deadline
+  * Personnel risks and difficulties with team members
+
+## Feature Work (10%-20% of time)
+
+* Balances individual contributor work with team coordination work, scaling up and down individual contributor work as needed to ensure team success
+* Acts as the QA backstop for sprint team projects, ensuring that sufficient QA is done before major launches/releases
```
Comment (pkarman) [inline docs/tech-lead-description.md]: QA backstop seems like a Project Management task to me

File: docs/tech-lead-description.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+* Ensures that knowledge and familiarity with all parts of the codebase are spread across the sprint team and the wider Caseflow team
+* Reviews PRs thoroughly
+* Pairs with other engineers on the sprint team
+
+## Project Management (50% of time)
+
+* Participates in collaborative project planning/explorations with product and design to develop and vet baseline requirements for new sprint team work
+* Is ultimately responsible for the technical design and architecture for all of the sprint team's projects, delegating as needed
+* Balances the team's work on new product requirements with technical debt/refactoring work to ensure engineering quality
+* Works with product and design to create tickets for the engineering team
```
Comment (pkarman) [inline docs/tech-lead-description.md]: this one seems like a duplicate of the first one?

File: docs/tech-lead-description.md (hunk 1 above)
Comment (sharonwarner) [reply docs/tech-lead-description.md]: Sounds good! Will remove that.

File: docs/tech-lead-description.md (hunk 2 above)
Comment (sharonwarner) [reply docs/tech-lead-description.md]: I'll clarify ticket creation as part of the first one.

**Pull request merged**